docs/index.md

```diff
@@ -0,0 +1,116 @@
+## Overview
+This guide walk-through the process of deploying an Sample Java application in Cloud Platform.
+
+Before working through the tutorial, we recommend reading following pages and assume you have all the required accounts.
+
+- https://hmcts.github.io/onboarding
+- https://hmcts.github.io/ways-of-working/
+
+This guide uses a pre-configured application from an template which can be deployed from [Backstage](https://backstage.platform.hmcts.net/). Application is deployed to AKS(kubernetes) cluster inside `labs` namespace in sbox/sandbox environment.
+
+## Steps
+
+1. Access [Backstage](https://backstage.platform.hmcts.net/create) to create a new github repo containing application template. Please make sure name of  Github repo  starts with `labs*`. This enables Jenkins build tool to organize the folder based on Github repo names.
+
+2. Create `Jenkinsfile_parameterized`  file in the repo with following contents.
+    ```
+        #!groovy
+        
+        @Library("Infrastructure")
+        
+        def type = "java"
+        def product = "labs"
+        def component = "mohanalatest"
+        
+        withPipeline(type, product, component) {
+        }
+3. Create Helm values file with following contents.
+
+   CFT: values.sandbox.template.yaml
+
+   SDS: values.sbox.template.yaml
+   
+    ```
+        java:
+          # Don't modify below here
+          image: ${IMAGE_NAME}
+          ingressHost: ${SERVICE_FQDN}
+4. Get the Pull Request reviewed and merged.
```

Comment:
Who is doing the review? How do they get it reviewed and merged?

Reply:
@timja are we responsible for reviewing PRs ?

Reply:
no there should not be any review